docs/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -44,7 +44,7 @@ In order to allow Kubelet to use containerd (using the CRI interface), configure
   ```bash
   $ sudo mkdir -p  /etc/systemd/system/kubelet.service.d/
   $ cat << EOF | sudo tee  /etc/systemd/system/kubelet.service.d/0-containerd.conf
-  [Service]                                                 
+  [Service]
```

Comment:
This change is not related to the problem you're fixing, please, drop it as well.